wavefront_sdk/client.py

```diff
@@ -56,7 +56,7 @@ class WavefrontClient(connection_handler.ConnectionHandler,
     def __init__(self, server, token, max_queue_size=50000, batch_size=10000,
                  flush_interval_seconds=5, enable_internal_metrics=True,
                  queue_impl=queue.Queue):
-        # pylint: disable=too-many-arguments,too-many-statements
+        # pylint: disable=R0913,R0917,too-many-statements
```

Comment:
You might consider commenting what R0913 and R0917 are. I know they probably mean too-many-arguments, but why are there two of them?

Reply:
Just use the literal instead of the code. Pylint directive is already a comment itself.